runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -285,6 +285,14 @@
 
   void setFasterCopy(Boolean fasterCopy);
 
+  @Description(
+      "Directory containing Flink YAML configuration files. "
```

Comment:
would be great if we can make it even more explicit that the configuration is only applied during JobGraph construction and doesn't affect the cluster configuration in any way.

nit: It would be great the description could be a constant that is used for both java and portable runner (to make sure these won't diverge by mistake)